KubeArmor/core/kubeArmor.go

```diff
@@ -647,10 +653,13 @@ func KubeArmor() {
 	}
 
 	if dm.K8sEnabled && cfg.GlobalCfg.Policy {
-		if dm.checkNRIAvailability() {
+		if cfg.GlobalCfg.UseOCIHooks {
+			go dm.HandleFile("/opt/output.json")
```

Comment:
need a better name for this JSON file. 
Also previous comment is not resolved I think, is it not required?